Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS base
 
 # PIPENV_DEV is set to true in the docker-compose allowing
 # local builds to install the dev dependencies
 ARG PIPENV_DEV=False
```

Comment:
Do we need this arg here? Or is it sufficient to only have the ARG on line 69?

Reply:
I don't _think_ we need it there. That's why I was looking into the subtle different between `ARG` and `ENV`. It doesn't look like anything in that stage is running `pipenv` commands, so I think it should only be needed in the final stage.

Reply:
I went ahead and remove this and just moved the comment down.